### DISCUSSION TITLE
Layer progress

### DISCUSSION THREAD

**Discussion by pgale** [KlipperScreen/KlipperScreen#420] [Ideas]
Love Klipperscreen - it's so good. One thing I'd like to see is the layer progress - current layer/total layers. Could go in the gap left by time left maybe?

**Comment by hapklaar** [comment]
Funny, was just thinking about this yesterday. This would indeed be a nice addition to the info shown on the printing screen!

**Reply by pgale** [reply]
yes - would be very useful.

**Comment by alfrix** [comment]
![imagen](https://user-images.githubusercontent.com/1247237/150758345-a4edde4a-dabd-49d7-bc7a-3b5207f31ec0.png)

**Reply by pgale** [reply]
So that displays layer progress in the text area under the file name? Would still be nice to have that as a main display item.

**Reply by thiagomagamoc** [reply]
> ![imagen](https://user-images.githubusercontent.com/1247237/150758345-a4edde4a-dabd-49d7-bc7a-3b5207f31ec0.png)
Is it possible to insert this in the Cura slicer?

**Comment by alfrix** [comment]
There is also an alternative way using SET_PRINT_STATS_INFO
https://klipperscreen.readthedocs.io/en/latest/Quicktips/#layer-progress
